static/app/views/settings/organizationAuditLog/auditLogList.tsx

```diff
@@ -270,53 +271,59 @@ function AuditLogList({
 
   return (
     <div>
-      <SettingsPageHeader title={t('Audit Log')} action={action} />
-      <PanelTable
-        headers={[t('Member'), t('Action'), t('IP'), t('Time')]}
-        isEmpty={!hasEntries && entries?.length === 0}
-        emptyMessage={t('No audit entries available')}
-        isLoading={isLoading}
-      >
-        {(entries ?? []).map(entry => {
-          if (!entry) {
-            return null;
-          }
-          return (
-            <Fragment key={entry.id}>
-              <UserInfo>
-                <div>{getAvatarDisplay(entry.actor)}</div>
-                <NameContainer>
-                  {addUsernameDisplay(entry.actor)}
-                  <AuditNote entry={entry} orgSlug={organization.slug} />
-                </NameContainer>
-              </UserInfo>
-              <FlexCenter>
-                <MonoDetail>{getTypeDisplay(entry.event)}</MonoDetail>
-              </FlexCenter>
-              <FlexCenter>
-                {entry.ipAddress && (
-                  <IpAddressOverflow>
-                    <Tooltip
-                      title={entry.ipAddress}
-                      disabled={entry.ipAddress.length <= ipv4Length}
-                    >
-                      <MonoDetail>{entry.ipAddress}</MonoDetail>
-                    </Tooltip>
-                  </IpAddressOverflow>
-                )}
-              </FlexCenter>
-              <TimestampInfo>
-                <DateTime dateOnly date={entry.dateCreated} />
-                <DateTime
-                  timeOnly
-                  format={is24Hours ? 'HH:mm zz' : 'LT zz'}
-                  date={entry.dateCreated}
-                />
-              </TimestampInfo>
-            </Fragment>
-          );
-        })}
-      </PanelTable>
+      {!organization.access.includes('org:write') ? (
+        <PermissionAlert />
```

Comment:
should we pass in `org:write` to this component?

Reply:
I think it might be a good idea, it should be a component higher in `/sentry/static/app/views/settings/organizationAuditLog/index.tsx`

Reply:
the current text is correct though 🤔

Reply:
If we go change the access scopes later, it should be in the index rather inside the separate components of each setting in case there is more than 1 component in a setting.

Reply:
maybe we should move `PermissionAlert` into the index. so you either have the permission and see the `AuditLogList` or you see the `PermissionAlert`